pom.xml

```diff
@@ -78,9 +78,11 @@
     </build>
 
     <modules>
-        <module>infinispan-spring-boot-starter-autoconfigure</module>
+        <module>infinispan-spring-boot-starter-embedded</module>
+        <module>infinispan-spring-boot-starter-remote</module>
         <module>infinispan-spring-boot-starter</module>
         <module>infinispan-spring-boot-starter-test-remote</module>
```

Comment:
Since we've done the split, we can squash some impl and test modules like `infinispan-spring-boot-starter-remote` and `infinispan-spring-boot-starter-test-remote`.